client/blocks/disconnect-jetpack-dialog/index.jsx

```diff
@@ -0,0 +1,123 @@
+				features.push( translate( '{{icon/}} Spam filtering', this.translateArgs( 'spam' ) ) );
+				break;
+
+			case 'premium':
+				features.push( translate( '{{icon/}} Daily, automated backups (unlimited storage)', this.translateArgs( 'history' ) ) );
+				features.push( translate( '{{icon/}} Daily, automated malware scanning', this.translateArgs( 'spam' ) ) );
+				features.push( translate( '{{icon/}} Priority WordPress and security support', this.translateArgs( 'chat' ) ) );
+				features.push( translate( '{{icon/}} 13Gb of high-speed video hosting', this.translateArgs( 'video' ) ) );
+				break;
+
+			case 'professional':
+				features.push( translate( '{{icon/}} Daily, automated backups (unlimited storage)', this.translateArgs( 'history' ) ) );
+				features.push(
+					translate( '{{icon/}} Daily, automated malware scanning with automated resolution',
+					this.translateArgs( 'spam' )
+				) );
+				features.push( translate( '{{icon/}} Priority WordPress and security support', this.translateArgs( 'chat' ) ) );
+				features.push( translate( '{{icon/}} Unlimited high-speed video hosting', this.translateArgs( 'video' ) ) );
+				features.push( translate( '{{icon/}} SEO preview tools', this.translateArgs( 'globe' ) ) );
+				break;
+		}
+
+		return features.map( ( freature, index ) => {
+			return (
+				<div key={ 'disconnect-jetpack-dialog__feature-' + index } className="disconnect-jetpack-dialog__feature">
+					{ freature }
+				</div>
+			);
+		} );
+	}
+
+	render() {
+		const { onStay, onDisconnect, isVisible, translate, isBroken, siteName } = this.props;
+		if ( isBroken ) {
+			return (
+				<Dialog isVisible={ isVisible } additionalClassNames="disconnect-jetpack-dialog" onClose={ onStay } >
+				<h1>{ translate( 'Disconnect Jetpack' ) }</h1>
+				<p className="disconnect-jetpack-dialog__highlight">
+					{ translate( 'WordPress.com has not been able to reach example.com for a while.' ) }
+				</p>
+				<div className="disconnect-jetpack-dialog__button-wrap">
+					<Button primary scary onClick={ onDisconnect }>{ translate( 'Remove Site', { context: 'Button' } ) }</Button>
+				</div>
+			</Dialog>
+			);
+		}
+
+		return (
+			<Dialog isVisible={ isVisible } additionalClassNames="disconnect-jetpack-dialog" onClose={ onStay } >
+				<h1>{ translate( 'Disconnect from WordPress.com?' ) }</h1>
+				<p className="disconnect-jetpack-dialog__highlight">
+					{
+						translate(
+							'By disconnecting %(siteName)s from WordPress.com you will no longer have access to the following:',
+							{ args: { siteName }, context: 'Jetpack Disconnect Dialog' }
+						)
+					}
+				</p>
+
+				{ this.planFeatures() }
+
+				<div className="disconnect-jetpack-dialog__button-wrap">
+					<Button onClick={ onStay }>{ translate( 'Stay Connected' ) }</Button>
+					<Button primary scary onClick={ onDisconnect }>{ translate( 'Disconnect', { context: 'Button' } ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Disconnect', { context: 'Sharing: Publicize disconnect button label'} )` *ES Score: 23*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/disconnect-jetpack-block)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `Jetpack: Action user takes to disconnect Jetpack site from .com`
- `Sharing: Publicize disconnect button label`

 Would it make sense to reuse one of the above?